### PR TITLE
Improve get_cams documentation

### LINKED ISSUE #1412: Improve ``get_cams`` documentation
**Describe the bug**
The geographical coverage of CAMS Radiation retrieved with the [``get_cams``](https://pvlib-python.readthedocs.io/en/stable/generated/pvlib.iotools.get_cams.html#pvlib.iotools.get_cams) function is not exactly ±66 degrees in latitude and longitude. Therefore, to clarify this, the word "approximate" should be added to the function documentation [here](https://github.com/pvlib/pvlib-python/blob/63c36db473ce9d7406bfb90526c37783515716c2/pvlib/iotools/sodapro.py#L59).

Also, the description has a typo - "Wordwide" should be corrected to "worldwide".

**Screenshots**
The actual CAMS Radiation coverage:
image

### PR DESCRIPTION
**Pull request by soma2000-lang**
- [x] Closes #1412
 - [x] I am familiar with the [contributing guidelines](https://pvlib-python.readthedocs.io/en/latest/contributing.html)
 - [x] Tests added
 - [x] Updates entries in [`docs/sphinx/source/reference`](https://github.com/pvlib/pvlib-python/blob/master/docs/sphinx/source/reference) for API changes.

### CODE REVIEW

**Comment by soma2000-lang** [comment]
@AdamRJensen  please look into this pr

**Comment by AdamRJensen** [comment]
@soma2000-lang I noticed that the stickler-ci check is failing, which means that there is some formatting check that is not passing, e.g., a line that is too longer than 79 characters. You can click the "Details" button to the right of the stickler-ci to see the error.

**Comment by AdamRJensen** [comment]
It happens sometimes that you fix a stickler error, only to break another rule. Btw, I thought it was nice when you spelled out approximately in full.

**Comment by soma2000-lang** [comment]
@AdamRJensen yes the error was showing that remove the trailing whitesapces ,I tried to solve that using
Find > Replace...
Find What: [ \t]+\n
Replace With: \n
Replace All

these steps but its still showing the same error .Can you please help me in this regard

**Comment by AdamRJensen** [comment]
@soma2000-lang The trailing white space refers to a space at the end of a line. The stickler-ci will show the warning just below the affected line, so that should help you find it another time.

File: pvlib/iotools/sodapro.py [hunk 1]
```diff
@@ -363,7 +360,7 @@ def read_cams(filename, integrated=False, label=None, map_variables=True):
 
     See Also
     --------
-    pvlib.iotools.parse_cams, pvlib.iotools.get_cams
+    pvlib.iotools.parse_cams,pvlib.iotools.get_cams
```
Comment (AdamRJensen) [inline pvlib/iotools/sodapro.py:363]: You can go ahead and put this space back since this was not the trailing white space.

**Review by AdamRJensen — COMMENTED** [review]
Otherwise it looks good :)

File: pvlib/iotools/sodapro.py [hunk 2]
```diff
@@ -55,11 +55,8 @@ def get_cams(latitude, longitude, start, end, email, identifier='mcclear',
     Access: free, but requires registration, see [1]_
 
     Requests: max. 100 per day
-
-    Geographical coverage: Wordwide for CAMS McClear and -66° to 66° in both
-    latitude and longitude for CAMS Radiation
-
-
+    Geographical coverage:worldwide for CAMS McClear and approximately -66° to
+    66° in both latitude and longitude for CAMS Radiation.
```
Comment (AdamRJensen) [inline pvlib/iotools/sodapro.py:59]:
```suggestion
    66° in both latitude and longitude for CAMS Radiation.

```

File: pvlib/iotools/sodapro.py [hunk 3]
```diff
@@ -55,11 +55,8 @@ def get_cams(latitude, longitude, start, end, email, identifier='mcclear',
     Access: free, but requires registration, see [1]_
 
     Requests: max. 100 per day
-
-    Geographical coverage: Wordwide for CAMS McClear and -66° to 66° in both
-    latitude and longitude for CAMS Radiation
-
-
+    Geographical coverage:worldwide for CAMS McClear and approximately -66° to
```
Comment (cwhanse) [inline pvlib/iotools/sodapro.py:58]:
```suggestion
    Geographical coverage: worldwide for CAMS McClear and approximately -66° to
```

**Review by cwhanse — APPROVED** [review]
thanks @soma2000-lang

**Comment by soma2000-lang** [comment]
@AdamRJensen @cwhanse  made the suugested changes

File: pvlib/iotools/sodapro.py [hunk 4]
```diff
@@ -55,11 +55,10 @@ def get_cams(latitude, longitude, start, end, email, identifier='mcclear',
     Access: free, but requires registration, see [1]_
 
     Requests: max. 100 per day
-
-    Geographical coverage: Wordwide for CAMS McClear and -66° to 66° in both
-    latitude and longitude for CAMS Radiation
-
-
+
```
Comment (AdamRJensen) [inline pvlib/iotools/sodapro.py:58]:
```suggestion

```

File: pvlib/iotools/sodapro.py [hunk 5]
```diff
@@ -55,11 +55,10 @@ def get_cams(latitude, longitude, start, end, email, identifier='mcclear',
...
     Requests: max. 100 per day
-
-    Geographical coverage: Wordwide for CAMS McClear and -66° to 66° in both
-    latitude and longitude for CAMS Radiation
-
-
+    
+    Geographical coverage: worldwide for CAMS McClear and approximately -66° to
+    66° in both latitude and longitude for CAMS Radiation.
+
```
Comment (AdamRJensen) [inline pvlib/iotools/sodapro.py:61]:
```suggestion

```

**Comment by AdamRJensen** [comment]
@soma2000-lang There still is two stickler-ci lines being flagged.

Notice that the stickler-ci tells you which line the errors are on, in this case "Check failure on **line 61** and the error is **blank line contains white space**.
image

Almost there :)

**Comment by cwhanse** [comment]
@soma2000-lang please add yourself to the Contributors section of docs/source/sphinx/whatsnew/v0.9.1.rst

**Pull request merged**